R/1.0_module_data_summary.R

```diff
@@ -0,0 +1,213 @@
+ui_data_summary = function(id) {
+  ns <- NS(id)
+  tags$div(
+    id = id, # not used, can be used to customize CSS behavior
+    class = "well",
+    tags$div(
+      class = "row",
+      tags$div(
+        class = "col-sm-9",
+        tags$label("Active Filter Summary - teal", class = "text-primary mb-4")
+      ),
+      tags$div(
+        class = "col-sm-3",
+        actionLink(
+          ns("minimise_filter_overview"),
+          label = NULL,
+          icon = icon("angle-down", lib = "font-awesome"),
+          title = "Minimise panel",
+          class = "remove pull-right"
+        )
+      )
+    ),
+    tags$div(
+      id = ns("filters_overview_contents"),
+      tags$div(
+        class = "teal_active_summary_filter_panel",
+        tableOutput(ns("table"))
+      )
+    )
+  )
+}
+
+#' @description
+#' Server function to display the number of records in the filtered and unfiltered data.
+#'
+#' @param id (`character(1)`)
+#'   `shiny` module instance id.
+#' @param filtered_teal_data (`reactive`) (`teal_data`) an output of `.make_teal_data()`
+#' @return `NULL`.
+srv_data_summary = function(id, filtered_teal_data) {
+  checkmate::check_class(filtered_teal_data, "reactive")
+  checkmate::check_class(filtered_teal_data(), "teal_data")
+
+  moduleServer(
+    id = id,
+    function(input, output, session) {
+      #shinyjs::hide(id = "teal-main_ui-filter-panel-overview") # this doesnt hide filter-panel-overiw from teal.slice YET
```

Comment:
I think this or `removeUI` needs to be called later after `srv_filter_panel` is called